### PR TITLE
Easier onboarding

### PR DESCRIPTION
**Pull request by chalkandpaste**
# Let's make onboarding easier

Core changes:

- Update README.md to make it easier to get started

Notes:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +13,48 @@
...
+To contribute to the Specular monorepo, it will be handy to have the Specular environment and
+toolchain set up. The Specular toolchain depends on:
+
+* go
+* Node with pNPM
+
+These can be installed on macOS with:
+
+```
+> brew install go pnpm
```
Comment (LEAFERx) [inline README.md:29]: pnpm should be installed with `corepack enable` after node installation

File: README.md [hunk 2]
```diff
@@ -12,6 +13,48 @@
...
+repository via git submodules including:
+
+* blockscout
+* go-ethereum
+* forge-std
+
+To collect and set up the required submodules, run:
+
+```
+> git submodule init
```
Comment (LEAFERx) [inline README.md:50]: `git submodule update --init --recursive`. or `git clone https://github.com/specularl2/specular --recursive`

File: README.md [hunk 3]
```diff
@@ -12,6 +13,48 @@
...
+> git submodule update --recursive
+```
+
+### Building the Project Locally
+
+TODO
```
Comment (siosw) [inline README.md:56]: not sure if you saw this but currently some documentation for building locally lives [here](https://github.com/SpecularL2/specular/tree/main/clients/geth/specular/README.md)

maybe we can link to it from the main readme to make it more obvious?

Comment (chalkandpaste) [reply README.md:56]: Thanks @siosw! Yes, I was going to either link it to that README or figure out something else. Running through that README what drove me to work on this, at the very least documenting things for myself, but hopefully of some help to others.

File: README.md (hunk 2 above)
Comment (chalkandpaste) [reply README.md:50]: 👍

File: README.md (hunk 1 above)
Comment (chalkandpaste) [reply README.md:29]: Good to know. I'll have to update my environment. I may figure this out before you respond, but do you know how to install with corepack with brew/apt?

Comment (chalkandpaste) [reply README.md:29]: NVM, I found instruction and will update my README changes to reflect shortly.

**Pull request closed without merging**